docs/basics.md

```diff
@@ -0,0 +1,203 @@
+**_These docs are for Marionette 3 which is still in pre-release. Some parts may
+not be accurate or up-to-date_**
+
+# Common Marionette Concepts
+
+This document covers the basic usage patterns and concepts across Marionette.
+This includes things like calling conventions, setting attributes, common option
+patterns etc.
+
+## Documentation Index
+
+* [Class-based Inheritance](#class-based-inheritance)
+  * [Value Attributes](#value-attributes)
+  * [Functions Returning Values](#functions-returning-values)
+* [Setting Options](#setting-options)
+  * [The `getOption` Method](#the-getoption-method)
+  * [The `mergeOptions` Method](#the-mergeoptions-method)
+
+## Class-based Inheritance
+
+Backbone and Marionette utilize the `_.extend` function to simulate class-based
+inheritance. All built-in classes, such as `Marionette.View`,
+`Marionette.Object` and everything that extend these provide an `extend` method
+for just this purpose.
+
+In the below example, we create a new pseudo-class called `MyView`:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var MyView = Mn.View.extend({});
+```
+
+You can now create instances of `MyView` with JavaScript's `new` keyword:
+
+```javascript
+var view = new MyView();
+```
+
+### Value Attributes
+
+When we extend classes, we can set a form of class attributes by setting new
```

Comment:
I'm having a difficult time reading this description.
Would this description help?
> When we extend classes, we can provide class attributes with specific values by defining them in the object we pass as the first parameter.

I would also note that the values are shallow copied over to the classes's prototype object. All object type values are passed by reference, so changing any attribute values will result in affect all class instances or extending classes.
I am not sure how to phrase all that in a way that isn't overwhelming, but I'd consider thinking of a way to do so.